README.md

```diff
@@ -618,9 +234,17 @@ npm start
 # Observe services tracing information with Jaeger
 
 After running the docker container, you can view the tracing information on Jaeger via http://localhost:16686/. You can select the service in the drop-down box as follows.
-![image info](drop-down-services-spans.png)
+![image info](images/drop-down-services-spans.png)
 
 Then click on `Find Traces` and you will be able to view spans of services in the gcp-microservice.
-![image info](tracing-spans.png)
+![image info](images/tracing-spans.png)
 
 For more information about observability in Ballerina, please visit [Observe Ballerina Programs](https://ballerina.io/learn/observe-ballerina-programs/).
+
```

Comment:
Did we decide to add combine the major features of Highlights.md with README.md as well? I think we did

Reply:
yes, I am deriving some info to add in the readme. Will update.

Reply:
Done.